frontend/src/Series/Index/Table/SeriesIndexRow.js

```diff
@@ -290,7 +290,7 @@ class SeriesIndexRow extends Component {
                     progress={progress}
                     kind={getProgressBarKind(status, monitored, progress)}
                     showText={true}
-                    text={`${episodeFileCount} / ${episodeCount}`}
+                    text={`${episodeFileCount} / ${episodeCount} (${totalEpisodeCount})`}
```

Comment:
This information is already shown in a separate column, it shouldn't be added here as well.